src/index.js

```diff
@@ -1,31 +1,56 @@
 import './style.css';
 
-const mylist = [
-  {
-    description: 'Watch coding Tutorial on YouTube',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Pickup the kids at school',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Buy  groceries',
-    completed: false,
-    index: 3,
-  },
+import script from '../modules/script.js';
 
-];
+// add amd remove
+const localData = script.retrieve();
+if (!localData) localStorage.setItem('mylist', '[]');
 
-const list = document.getElementById('list');
+const display = () => {
+  const storeData = script.retrieve();
 
-mylist.sort((a, b) => a.index - b.index);
-mylist.forEach((value) => {
-  const li = document.createElement('li');
-  li.innerHTML = `
-        ${value.description}<i class="fa-solid fa-ellipsis-vertical"></i>
+  const list = document.getElementById('list');
+  list.innerHTML = '';
+  storeData.forEach((value) => {
+    // Creating list of to-do
+    const li = document.createElement('li');
+    li.innerHTML = `
+    <input type="checkbox">
+    <input class="text" type="text" value="${value.description}"/> 
+    <i class="fa-solid fa-ellipsis-vertical"></i>
     `;
-  list.appendChild(li);
-});
\ No newline at end of file
+    const removeButton = document.createElement('div');
+    removeButton.innerHTML = '<i class="fa-regular fa-trash-can"></i>';
+    removeButton.addEventListener('click', () => {
+      script.remove(value.id);
+      display();
+    });
+    li.appendChild(removeButton);
+    list.appendChild(li);
+  });
+
+  const span = document.querySelectorAll('.text');
+  span.forEach((btn, index) => {
+    btn.addEventListener('keyup', () => {
+      const test = script.retrieve();
+      test[index].description = btn.value;
+      script.save(test);
+    });
+  });
+};
+
+const renderList = () => {
+  const form = document.getElementById('form');
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const storeData = script.retrieve();
+    const input = form.text.value;
+    const completed = false;
+    const id = storeData.length;
+
+    script.add(input, completed, id);
+    display();
+    form.text.value = '';
+  });
+};
+renderList();
```

Comment:
- [ ] Please, I observed that if I refresh the browser after performing operation on your app, the `todos` in the `localstorage` were not displayed to the UI. Kindly fix this. 👍 

<img width="879" alt="Untitled" src="https://github.com/techmoves/todo-list/assets/103386183/5bbd8a4c-20dc-4e4b-beeb-98620fd0af52">

Reply:
hi [OyePriscilla](https://github.com/OyePriscilla)

when you add a new todos they show-up on the UI

Reply:
@techmoves Yes, they do.